packages/wonder-blocks-core/components/clickable.js

```diff
@@ -0,0 +1,141 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
```

Comment:
Before we add this to core - we should ask if we want this in core (since otherwise will require a version change later to move it). I guess if we ever wanted to move `getClickableBehavior` out of core and this into a "clickable" package we'd have to bump the version anyway.

Reply:
Good call.  Maybe we could create a `wonder-blocks-clickable` package.  We can move `ClickableBehavior` over that package but still have `wonder-blocks-core` export `ClickableBehavior` though I think we'd want to eventually drop that export and have webapp just use `Clickable` and have `ClickableBehavior` be completely private.